File: src/main/java/com/gmail/borlandlp/minigamesdtools/creator/AbstractDataProvider.java
package com.gmail.borlandlp.minigamesdtools.creator;

import java.util.Set;

public interface AbstractDataProvider {
    boolean contains(String key);
    Object get(String key);
    void set(String key, Object value);
    void remove(String key);
    Set<String> getKeys();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/creator/AbstractDataProvider.kt
package com.gmail.borlandlp.minigamesdtools.creator

interface AbstractDataProvider {
    operator fun contains(key: String): Boolean
    operator fun get(key: String): Any
    operator fun set(key: String, value: Any)
    fun remove(key: String)
    val keys: Set<String>
}